crates/rooch-rpc-client/src/wallet_context.rs

```diff
@@ -60,7 +68,21 @@ impl WalletContext {
         address_mapping.extend(addresses::rooch_framework_named_addresses());
 
         //TODO support account name alias name.
-        if let Some(active_address) = client_config.active_address {
+        if let Some(active_address) = &client_config.active_address {
+            let active_address = if !keystore.contains_address(active_address) {
+                //The active address is not in the keystore, maybe the user reset the keystore.
+                //We auto change the active address to the first address in the keystore.
```

Comment:
When the active address is not in the keystore, the order here should be to use the active address in the keystore first. If there is no active one, then choose the first address.